src/index.css

```diff
@@ -1,13 +1,280 @@
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.btn:active {
+  box-shadow: none;
+}
+
+.warningMessage {
+  color: #f72585;
+  padding: 10px;
+  height: 35px;
+  font-weight: 400;
+}
+
+form {
+  position: relative;
+}
+
+
+/* Modal Window Styling */
+.modalWindow {
+  background-color: #fff;
+  padding: 30px;
+  border-radius: 20px;
+  position: absolute;
+  width: 30%;
+  box-shadow: 3px 5px 15px 11px rgba(0, 0, 0, 0.1);
+  top: 80px;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.modalBtn {
+  margin-top: 20px;
+}
+
+/* Grocery Item styling */
+.flexContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+span {
+  color: #fff;
+  background-color: #c19bff;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 1.3rem;
+  height: 1.3rem;
+  margin-left: 10px;
+  font-weight: 500;
+}
+
+.totalItem {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.clearList {
+  background-color: #f72585;
+  margin: 0;
+}
+
+.listContainer {
+  margin: 10px auto 20px auto;
+  text-align: center;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+li {
+  font-weight: 300;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  padding: 10px 10px;
+  margin: 3px 0;
+}
+
+.rightItems {
+  display: flex;
+}
+
+img {
+  width: 70px;
+  height: 70px;
+  display: block;
+  object-fit: cover;
+  margin-right: 15px;
+}
+
+.itemName {
+  text-transform: uppercase;
+}
+
+.btnContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.removeItem {
+  align-self: flex-end;
+  margin-bottom: 40px;
+}
+
+.updateBtns {
+  display: flex;
+  align-items: center;
+}
+
+.icon {
+  color: #9a5cff;
+}
+
+.changeAmount {
+  font-size: 0.5rem;
+  padding: 5px 6px;
+  border: 1px solid #9a5cff;
+  border-radius: 50%;
+  transition: 0.3s;
+}
+
+.changeAmount:hover,
+.changeAmount:focus {
+  background-color: #c19bff;
+  color: white;
+  border: 1px solid #c19bff;
+}
+
+.itemAmount {
+  width: 20px;
+}
+
+/* Footer styling */
+footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  text-align: center;
+  background-color: #c19bff;
+  padding: 3px 0;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+}
+
+a:hover,
+a:focus {
+  color: white;
+}
+
+/* Media Queries */
+@media(max-width: 780px){
+  .modalWindow {
+    width: 70%;
+  }
+}
+@media (max-width: 555px) {
+  .wrapper {
+    width: 95%;
+  }
+
+  input {
+    width: 74%;
+  }
+}
+
+@media (max-width: 430px) {
```

Comment:
I checked it with my phone and it looks sick 🏆 🍹 🍸

Reply:
yay! 😊